node/service/src/lib.rs

```diff
@@ -686,6 +695,38 @@ pub fn westend_new_full(
 	Ok((components, client, FullNodeHandles))
 }
 
+/// Create a new Polkadot service for a full node.
+#[cfg(feature = "full-node")]
+pub fn origami_new_full(
```

Comment:
This should probably be in `service-new`, because that is the service that uses the `overseer`.